docs/user-guide/src/bmo/automatic_secure_boot.md

```diff
@@ -0,0 +1,30 @@
+# Automatic secure boot
+
+The automatic secure boot feature allows enabling and disabling UEFI (Unified Extensible Firmware Interface) secure boot when provisioning a host. This feature requires supported hardware and compatible OS image. The current hardwares that support enabling UEFI secure boot are `iLO driver`, `iRMC driver` and `Redfish driver`.
+
+Check also
+
+- [Ironic UEFI secure-boot](https://docs.openstack.org/ironic/latest/admin/security.html#uefi-secure-boot-mode)
```

Comment:
"secure boot" (without a dash, it's not a compound word). same below

Reply:
Done previously